mainnet/README.MD

```diff
@@ -0,0 +1,220 @@
+## State Extraction and Creation
```

Comment:
nit: file extension consistency `.MD` `.md`

Reply:
removed in [b501657](https://github.com/ar-io/ar-io-network-process/pull/388/commits/b50165766471e76ce85d798e388178ccc591bb1d)